Classes/Tca/ContainerConfiguration.php

```diff
@@ -89,6 +92,12 @@ public function __construct(
         $this->label = $label;
         $this->description = $description;
         $this->grid = $grid;
+        if ((GeneralUtility::makeInstance(Typo3Version::class))->getMajorVersion() === 12) {
```

Comment:
"=== 12" vs. "> 11" :)

Reply:
hm, yes, but the filename is named `Partials12` , maybe we shoult change this when dropping v10 Support to `majorVersion === 11 then: Partials11` (but maybe Cores Partial File will change in 13 again, and we have to adapt our Partial again?, Nothing can be said, for this I fix the Condition and the Partial Name to `12`